.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+* **Please check if the PR fulfills these requirements**
```

Comment:
Is this for the PR creator of for the reviewer? It is not clear here, if it is for the Reviewer, a "do not change" might be interesting

Reply:
It's for PR creators.